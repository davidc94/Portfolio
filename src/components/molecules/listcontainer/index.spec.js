import React from "react"
import renderer from "react-test-renderer"
import ListContainer from "./index";

describe("ListContainer", () => {
  it("renders as expected", () => {
    expect(renderer.create(<ListContainer />).toJSON()).toMatchSnapshot();
  })
})