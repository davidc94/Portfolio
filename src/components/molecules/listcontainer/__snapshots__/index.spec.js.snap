// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListContainer renders as expected 1`] = `
<div
  className="sc-bwzfXH dgmRxW"
>
  <li
    className="sc-bdVaJa imCgTo"
    size="large"
  >
    This is a list
  </li>
  <li
    className="sc-bdVaJa imCgTo"
    size="large"
  >
    This is a list
  </li>
  <li
    className="sc-bdVaJa imCgTo"
    size="large"
  >
    This is a list
  </li>
</div>
`;
